datahub-classify/setup.py

```diff
@@ -14,10 +14,12 @@ def get_long_description():
 
 
 base_requirements = {
+    "vininfo==1.7.0",
+    "schwifty==2022.9.0",
+    "python-stdnum==1.17",
+    "ipaddress==1.0.23",
```

Comment:
we should use `>=` requirements where possible

Reply:
Currently, the code is tested using these specific versions, that's why we pinned it to a specific version. In the future, we will take care of this.

Reply:
I think it would be good to do this now - mayuri already ran into issues because the requirements here are too strict, so we should fix it in this PR.

Reply:
Used ">=" instead of pinning libraries to specific version.